independent-publisher-connectors/HubSpot CRM/apiDefinition.swagger.json

```diff
@@ -0,0 +1,6272 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",

```

Comment:
@troystaylor Can you please provide valid title for connector.

Documentation - [Title](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#title)